#!/usr/bin/env python
PACKAGE = "lane_mapping"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("disc_factor",    double_t,    0, "An Integer parameter", 0.0009525,  0.0005, 0.0010)
gen.add("scaling_factor",    int_t,    0, "An Integer parameter", 100000,  100000, 1000000)
gen.add("angle_increment",    double_t,    0, "An Integer parameter", 0.11894988298416137,  0.008750975131988525, 1)
gen.add("range_size",    int_t,    0, "An Integer parameter", 180,  1, 360)
# gen.add("int_param",    int_t,    0, "An Integer parameter", 50,  0, 100)

exit(gen.generate(PACKAGE, "lane_mapping", "parameters"))